server/mailet/integration-testing/src/main/java/org/apache/james/mailets/TemporaryJamesServer.java

```diff
@@ -140,7 +140,8 @@ public static Builder builder() {
         "recipientrewritetable.xml",
         "usersrepository.xml",
         "smime.p12",
-        "smime_cert_keystore");
+        "smime_cert_keystore",
+        "trusted_certificate.pem");
```

Comment:
Please do 2 tests: one with pem, one with keystore

Reply:
added two different test classes (one with keystore, one with pem) for SMIMECheckSignature